controllers/rest/utils.go

```diff
@@ -268,6 +268,32 @@ func SplitFilter(filter string) *FilterProperties {
 	return property
 }
 
+func SplitHeaders(header string) *HeaderProperties {
```

Comment:
I like the idea of this helper method. I think it could be a bit more abstract to allow for setting any key value pair that is sent as query string variables in that format. Maybe don't use the word "header" and don't hard code it to look for "_headers" (maybe the param key is passed in as another parameter)